packages/@aws-cdk/aws-codepipeline-actions/lib/servicecatalog/deploy-action.ts

```diff
@@ -0,0 +1,99 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { Role, PolicyStatement, ManagedPolicy } from '@aws-cdk/aws-iam';
```

Comment:
We generally don't use per-item imports for packages other that `@aws-cdk/core`. So, this should be:

```suggestion
import * as iam from '@aws-cdk/aws-iam';
```

Like it's above for `codepipeline`.